### PR TITLE
docs: improve manager documentation

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes:

- Rewrite some sentences to be clearer
- Fix some "one sentence per line" problems
- Standardize on `regular expression(s)` instead of using `value(s)`

## Context:

Helps with issue #7626.

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added unit tests, or
- [ ] Unit tests + ran on a real repository

### CODE REVIEW

File: docs/usage/modules/manager.md [hunk 1]
```diff
@@ -85,16 +87,17 @@ To disable all managers within a language like `python`, do this:
 }
 ```
 
-Note: Only languages declared by a Renovate manager can be supported, so please verify first.
+Only languages declared by a Renovate manager can be supported, so please verify first.
```
Comment (HonkingGoose) [inline docs/usage/modules/manager.md:90]:
```markdown
Only languages declared by a Renovate manager can be supported, so please verify first.
```

Where should a reader check what languages are supported?

**Comment by HonkingGoose** [comment]
I'm not happy with the current line 35:

```markdown
If the default `fileMatch` value for a manager does not match against one of your relevant files, you can _extend_ the existing value(s) by configuring a manager's `fileMatch` like in this example:
```

This sentence really should be simplified in some way.

I think what we're trying to say here is: "If you don't get updates for a file you would expect to get updates for, extend the existing regular expressions with the `fileMatch` array."

---

I also noticed that we're using `value` and `regular expression`, which are basically the same in this context?
The `value` is the `regular expression` array?
Does anybody have a good idea on simplifying this?

File: docs/usage/modules/manager.md (hunk 1 above)
Comment (viceice) [reply docs/usage/modules/manager.md:90]: see https://docs.renovatebot.com/modules/manager/#supported-managers
 on the left are the `languages`, `other` means no language

**Comment by viceice** [comment]
> I'm not happy with the current line 35:
> 
> ```gfm
> If the default `fileMatch` value for a manager does not match against one of your relevant files, you can _extend_ the existing value(s) by configuring a manager's `fileMatch` like in this example:
> ```
> 
> This sentence really should be simplified in some way.
> 
> I think what we're trying to say here is: "If you don't get updates for a file you would expect to get updates for, extend the existing regular expressions with the `fileMatch` array."

docs for `fileMatch` are [here](https://docs.renovatebot.com/configuration-options/#filematch)


renovate will merge the predefined regexes with the user defined regexes

File: docs/usage/modules/manager.md (hunk 1 above)
Comment (HonkingGoose) [reply docs/usage/modules/manager.md]: Thanks @viceice, I've added a link to that section in the document. :smile:

**Comment by HonkingGoose** [comment]
I'm marking this as ready for review.

I still have a question though:

We're using `value` and `regular expression` somewhat interchangeably.
Maybe we should standardize on saying either `value` or `regular expression`?
Or do you have some other idea to clear this up?

**Comment by viceice** [comment]
I think we should stick to regular expression to be clear there

File: docs/usage/modules/manager.md [hunk 2]
```diff
@@ -15,12 +15,12 @@ The goal of Renovate is to detect and maintain all third party dependencies in y
 ### File Matching
 
 Most managers have a default `fileMatch` array.
-`fileMatch` is an array of Regular Expression strings used to match against the repository file list.
+The `fileMatch` array contains Regular Expression strings that match against the repository file list.
```
Comment (JamieMagee) [inline docs/usage/modules/manager.md:18]:
```suggestion
The `fileMatch` array contains regular expression strings that match against the repository file list.
```

**Pull request merged**